pinot-core/src/main/java/com/linkedin/pinot/core/io/reader/impl/v1/BaseChunkSingleValueReader.java

```diff
@@ -129,8 +88,7 @@ public VarByteReaderContext createContext() {
    * @param context Reader context
    * @return Chunk for the row
    */
-  private ByteBuffer getChunkForRow(int row, VarByteReaderContext context)
-      throws IOException {
+  protected ByteBuffer getChunkForRow(int row, ChunkReaderContext context) {
```

Comment:
Hmmm, I felt the current one was more clear. Anyway, since you left it up to me ;-) I'll keep it.